source/docs/fine_grain_log.md

```diff
@@ -40,7 +40,25 @@ Runtime update of Fine-Grain Logger is supported with administration interface,
 3. `POST /logging?paths=file_path1:level1,file_path2:level2...`: Change log level of multiple file paths at once;
 4. `POST /logging?level=<level>`: Change levels of all loggers.
 
-Users can view and change the log level in a granularity of file in runtime through admin page. Note that `file_path` is determined by `__FILE__` macro, which is the path seen by preprocessor.
+Users can view and change the log level in a granularity of file in runtime through admin page.
```

Comment:
s/in a granularity of file in runtime through admin page./by file granularity at runtime through the admin page./

Reply:
Done.